### PR TITLE
docs(cbr): add some fields for CBR

### PR DESCRIPTION
**Pull request by Zhukun-Huawei**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
=== RUN   TestAccCBRV3Policy_basic
=== PAUSE TestAccCBRV3Policy_basic
=== CONT  TestAccCBRV3Policy_basic
--- PASS: TestAccCBRV3Policy_basic (40.89s)
PASS

coverage: 1.6% of statements in ../../../terraform-provider-flexibleengine/...
```
```
=== RUN   TestAccCBRV3Policy_retention
=== PAUSE TestAccCBRV3Policy_retention
=== CONT  TestAccCBRV3Policy_retention
--- PASS: TestAccCBRV3Policy_retention (39.38s)
PASS
coverage: 1.6% of statements in ../../../terraform-provider-flexibleengine/...
```
```
=== RUN   TestAccCBRV3Policy_replication
=== PAUSE TestAccCBRV3Policy_replication
=== CONT  TestAccCBRV3Policy_replication
    acceptance.go:109: Skipping the replication policy acceptance tests.
--- SKIP: TestAccCBRV3Policy_replication (0.00s)

Test ignored.
PASS

coverage: 1.1% of statements in ../../../terraform-provider-flexibleengine/...
```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
/retitle docs(cbr): add some fields for CBR

File: flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go [hunk 1]
```diff
@@ -241,6 +247,7 @@ resource "flexibleengine_cbr_policy" "test" {
     daily   = 10
     weekly  = 10
     monthly = 1
+	full_backup_interval = -1
```
Comment (chengxiangdong) [inline flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go:250]:
```suggestion

    full_backup_interval = -1	
```

File: flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go [hunk 2]
```diff
@@ -264,6 +271,7 @@ resource "flexibleengine_cbr_policy" "test" {
     weekly  = 20
     monthly = 6
     yearly  = 1
+	full_backup_interval = 5
```
Comment (chengxiangdong) [inline flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go:274]: ditto

**Review by chengxiangdong — COMMENTED** [review]
LGTM, just needs to fix some nits.

File: flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go (hunk 2 above)
Comment (Zhukun-Huawei) [reply flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go]: done

File: flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go (hunk 1 above)
Comment (Zhukun-Huawei) [reply flexibleengine/acceptance/resource_flexibleengine_cbr_policy_test.go]: done

**Pull request merged**